docs/pages/access-controls/guides/moderated-sessions.mdx

```diff
@@ -4,6 +4,12 @@ description: Moderated Sessions
 h1: Moderated Sessions
 ---
 
+<Details title="Details title" min="8.3" opened>
```

Comment:
I suggest condensing the text into something like,

```markdown
Moderated Sessions is only available in Teleport 8.3.0 and above. If you are using a previous version of Teleport, you must specify `version: v4` or earlier for role resources.
```

Reply:
Don't forget to update the title!

Reply:
Good catch